### DISCUSSION TITLE
What should me do in first step when i pip this model

### DISCUSSION THREAD

**Discussion by MrkLong** [MatthiasValvekens/pyHanko#216] [Q&A] [answered]
I just finished downloading it and I don't know where to start. I'm a novice programmer

**Comment by MatthiasValvekens** [comment, marked as answer]
Well, that question is impossible for any of us to answer if you don't specify what you want to achieve ;).

Either way, [the documentation](https://pyhanko.readthedocs.io/en/latest/) should provide you with enough information to get started.

**Reply by MrkLong** [reply]
I'm wondering how to modify the --field parameter to show the default appearance on the output.pdf, obviously I don't know much

**Comment by MrkLong** [comment]
When I configured in pkcs11 and found that there were no libsofthsm2.so files, so I searched online myself and downloaded softHSM2, and finally the module-path was "C:SoftHSM2libsofthsm2-x64.dll", and then I started running "pyhanko sign addsig pkcs11 --p11-setup test-setup input.pdf output.pdf", terminal error "2023-02-08 08:56:26,961 - pyhanko.cli - ERROR - PKCS#11 error
Error: PKCS#11 error: [PKCS11Error] No token matching criteria TokenCriteria(label='testrsa', serial=None) found"
What is the reason for this, I feel that it is not a system problem, my system is Windows 10, while my friend Apple system does not have so many problems
